src/lesson5/NetSalaryCalculator.java

```diff
@@ -0,0 +1,131 @@
+package lesson5;
+
+public class NetSalaryCalculator {
+
+    public static void main(String[] args) {
+
+        int grossSalary = 1050000;
+        boolean isITField = true;
+
+        double incomeTax = incomeTaxOfGrossSalary(grossSalary, isITField);
+        System.out.println("The income tax of the given salary is " + incomeTax);
+
+        boolean isParticipating = true;
+        boolean isJoinedVoluntarily = false;
+        double socialFee = socialTaxOfGrossSalary(grossSalary, isParticipating, isJoinedVoluntarily);
+        System.out.println("The social fee of the given salary is " + socialFee);
+
+        double stampFee = stampTaxOfGrossSalary(grossSalary);
+        System.out.println("The stamp fee of the given salary is " + stampFee);
+
+        double netSalary = grossSalary - incomeTax - socialFee - stampFee;
+        System.out.println("Net salary is " + netSalary);
+
+    }
+
+    /**
+     *
+     * @param grossSalary The given gross salary
+     * @return true if the grossSalary is lower than the defined minimal salary, false otherwise
+     */
+    public static boolean isMinimalSalary(int grossSalary) {
+
+        return grossSalary < 68000 ? true : false;
```

Comment:
Great idea to separate method which calculates the if the salary is minimal also it would be better to have a separate variable for minSalary as it may change. It would add more readability to your code as 68.000 is just a hardcoded number.